README.md

```diff
@@ -27,5 +27,8 @@ If your username is in the form of UPN (user@contoso.com), you can hit the commo
 - Place table fields on new lines - When CSL table files are written to a file target, each field is placed on a new line. This can make it easier to diff changes in a pull request.
 - Generated ".create-merge table" commands instead of ".create table" - When CSL table files are written to a file target, the command will be ".create-merge table" if this is checked. If it is unchecked, the command will be ".create table".
 
+### Files
+- Use legacy `.csl` file extensions - Kusto originally used files with `.csl` file extensions. This extension has since been deprecated and replaced with the `.kql` file extension. This setting tells synckusto which file extension should be used for reading _and_ writing all files. The default state is unchecked, which means synckusto will consume and emit files with the `.kql` file extension.

```

Comment:
Thanks for the change, but I think the readme needs to be updated too?

"The default state is checked, which means synckusto will consume and emit files with the .csl file extension."